service-python/assessclaimdc7101/src/lib/continuous_medication.py

```diff
@@ -25,9 +13,9 @@ def continuous_medication_required(request_body):
 
   veterans_medication = request_body["evidence"]["medications"]
   for medication in veterans_medication:
-    medication_display = medication["description"]
-    for keyword in [x.lower() for x in hypertension_medications]:
-      if (keyword in medication_display.lower()):
-        relevant_medications.append(medication)
+    if medication["status"].lower() == "active":
+      relevant_medications.append(medication)
   
```

Comment:
I would break here. Otherwise you might end up adding a medication twice as in 200 ACTUAT Albuterol 0.09 MG/ACTUAT Metered Dose Inhaler

Reply:
I am iterating through the medications, If it meets the condition it is added to the list. I do not see how it will be added twice, the keyword list and conditional logic were removed for now

Reply:
Because two keywords can be in the medication description like in the example I gave that comes from the sandbox ("200 ACTUAT Albuterol 0.09 MG/ACTUAT Metered Dose Inhaler" ). IN this case both Albuterol and Inhal gets hit. SInce you do not break after you find the first one the medication is added twice.

Reply:
Why did you remove the logic? I thought we need the logic to match va.gov. You just need to fix it.

Reply:
The medication list is from the hypertension prototype. My understanding is that in va.gov there is no medication list for hypertension.